pytorch_lightning/plugins/training_type/sharded.py

```diff
@@ -28,10 +28,15 @@
 
 class DDPShardedPlugin(DDPPlugin):
 
+    REDUCE_BUFFER_SIZE_DEFAULT = 2**23
```

Comment:
```suggestion
    _REDUCE_BUFFER_SIZE_DEFAULT = 2**23  # 8 MiB 
```

Reply:
should be # of parameters per bucket

I would use 8M